packages/core/middleware/nuxt/index.js

```diff
@@ -1,5 +1,9 @@
-const createProxyEndpoint = require('./../middleware');
+const { createServer } = require('@vue-storefront/middleware');
 
-module.exports = function VueStorefrontMiddleware (moduleOptions) {
-  this.addServerMiddleware(createProxyEndpoint(moduleOptions, this.options));
+module.exports = function VueStorefrontMiddleware () {
+  const { integrations } = require(this.nuxt.options.rootDir + '/middleware.config.js');
```

Comment:
Wdyt about wrapping this line in try...catch and adding useful log for potential `middleware.config.js` without `.integrations`?

Reply:
I have to write all handling errors, such as handling 404, 500 etc. but that's for another issue - don't want to make this PR bigger and bigger